features/step_definitions/networking.rb

```diff
@@ -989,3 +989,31 @@
   raise "OVN is not running correctly! Check one of your ovnkube-node pod" unless desired_ovnkube_node_replicas == available_ovnkube_node_replicas && available_ovnkube_node_replicas != 0
   raise "OVN is not running correctly! Check one of your ovnkube-master pod" unless desired_ovnkube_master_replicas == available_ovnkube_master_replicas && available_ovnkube_master_replicas != 0
 end
+
+Given /^I store multicast keyval according to the networktype in the#{OPT_SYM} clipboard$/ do |cb_keyval|
```

Comment:
Can we rename the step to `I store multicast annotation suitable for current network type in the#{OPT_SYM} clipboard`?

Reply:
@akostadinov  create Given /^I enable multicast for the "(.+?)" namespace now.
New log from testing ovn cluster:
https://privatebin-it-iso.int.open.paas.redhat.com/?b29383740e18b36d#c2jFDsUgg39zYNyBJaRADG8YUYYCtZcYDXSuRJIPnTI=

New log from testing sdn cluster:
https://privatebin-it-iso.int.open.paas.redhat.com/?538718d77b3cbbdc#23kXoYOHTbWMQ2LkQyzMROjkJFq/uxcJ754Px7qZf30=